snmp/metadata.csv

```diff
@@ -89,6 +89,8 @@ snmp.fgFwPol6ByteCount.rate,gauge,,packet,second,[Fortinet FortiGate] The number
 snmp.ifAdminStatus,gauge,,,,[Generic router] [F5 BIG-IP] [Cisco c3850] [Cisco Nexus] [Cisco ASA 5525] The desired state of the interface.,0,snmp,
 snmp.ifSpeed,gauge,,,,"[Generic router] An estimate of the interface's current bandwidth in bits per second, or the nominal bandwidth.",0,snmp,
 snmp.ifHighSpeed,gauge,,,,"[Generic router] An estimate of the interface's current bandwidth in units of 1,000,000 bits per second, or the nominal bandwidth.",0,snmp,
+snmp.bandwidthInUsage.rate,rate,,,,"[Generic router] The rate of used received bandwidth in b/s.",0,snmp,
+snmp.bandwidthOutUsage.rate,rate,,,,"[Generic router] The rate of used sent bandwidth in b/s.",0,snmp,
```

Comment:
```suggestion
snmp.bandwidthInUsage.rate,gauge,,,,"[Generic router] The rate of used received bandwidth in b/s.",0,snmp,
snmp.bandwidthOutUsage.rate,gauge,,,,"[Generic router] The rate of used sent bandwidth in b/s.",0,snmp,
```

`rate` submission type are reported as in-app `gauge` type:

https://github.com/DataDog/datadog-agent/blob/master/pkg/metrics/rate.go